src/scene_skill.cpp

```diff
@@ -46,6 +46,11 @@ void Scene_Skill::Start() {
 	skill_window->SetHelpWindow(help_window.get());
 }
 
+void Scene_Skill::Continue(SceneType prev_scene) {
```

Comment:
prev_scene is unused here, simply delete the param name

Reply:
Well, the `Continue` function expects that the `prev_scene` parameter is present even if it is unused, otherwise compiling fails. Or did I miss something?

Reply:
He means leave the parameter, but delete it's name. Otherwise you get a compiler warning about unused variables

Reply:
So this means to change this to `void Scene_Skill::Continue(SceneType)` in `scene_skill.cpp` while leaving this in `scene_skill.h` unchanged? (I'm probably asking too much dumb questions)

Reply:
yeah keep in h, delete in cpp

Reply:
Fixed.